mobx/lib/src/api/annotations.dart

```diff
@@ -1,14 +1,11 @@
 /// Internal class only used for code-generation with `mobx_codegen`.
 ///
 /// During code-generation, this type is detected to identify Store class with generated toString() method
-class MakeToString {
-//  unnamed constructor to allow parameters in Store annotation and prevent useless code refactoring for mobx users
-  const MakeToString._();
+class MakeStoreConfig {
+  const MakeStoreConfig({this.makeToString = false});
```

Comment:
field should be called `toString` instead of `makeToString`

Reply:
> toString is an override so I named the parameter makeToString.
When using toString everything is falling apart.

Reply:
how about a more explicit `hasToString`? It's a boolean field and `hasXXX`, `isXXX` is standard practice.